### PR TITLE
[IMP] theme_vehicule: improve theme

### PR DESCRIPTION
**Pull request by xlu-odoo**
task-4178091
Part of task-4177975

### CODE REVIEW

File: theme_vehicle/static/description/theme_vehicle.svg
Comment (qsm-odoo) [inline theme_vehicle/static/description/theme_vehicle.svg:1]: @xlu-odoo I push-forced the branch with a rebase+conflict resolution and was planning to merge but... the SVG here is just broken :grimacing: misaligned title, etc.

Could you adapt (don't forget: I pushed the branch, fetch it first)?

**Review by qsm-odoo — CHANGES_REQUESTED** [review]

File: theme_vehicle/static/description/theme_vehicle.svg
Comment (xlu-odoo) [reply theme_vehicle/static/description/theme_vehicle.svg:1]: Thanks, updated

File: theme_vehicle/static/src/scss/primary_variables.scss [hunk 1]
```diff
@@ -104,11 +101,11 @@ $o-website-values-palettes: (
     (
         'color-palettes-name': 'vehicle-1',
 
-        'font': 'Fira Code',
-        'headings-font': 'Fira Sans',
-        'navbar-font': 'Fira Sans',
```
[the remark this answers is not in the document]
Comment (xlu-odoo) [reply theme_vehicle/static/src/scss/primary_variables.scss:109]: Honestly we coud have keep them so they are still available in the list, but I got your point, I removed them and put the new ones on top.

File: theme_vehicle/views/customizations.xml [hunk 2]
```diff
@@ -283,57 +299,49 @@
...
     </xpath>
-    <xpath expr="(//p[hasclass('card-text')])[3]" position="after">
-        <p><a href="#" class="btn btn-primary btn-block mb-0 o_cc o_cc4">Configure</a></p>
+    <xpath expr="(//p[hasclass('card-text')])[3]" position="replace" mode="inner">
+        Designed for the thrill-seeker, the Koran GT brings race-inspired performance to the streets. Precision-tuned for speed, it promises an driving experience.
     </xpath>
     <xpath expr="(//p[hasclass('card-text')])[3]" position="after">
-        <span class="d-inline-block mb-4"><b>From $25,000</b></span>
+        <strong>From $25,000</strong>
+        <p><a href="#" class="btn btn-primary mt-3">Configure</a></p>
```
[the remark this answers is not in the document]
Comment (xlu-odoo) [reply theme_vehicle/views/customizations.xml]: OK thanks @xO-Tx , have updated a few of the new page templates things.
s_three_columns: price and button are only set to be displayed via configurator
s_title: o_cc & custom padding only set to be displayed via configurator 

( We will revisit those new page templates for all theme after in second phase)

File: theme_vehicle/views/new_page_template.xml [hunk 3]
```diff
@@ -115,10 +115,17 @@
...
+        <attribute name="class" remove="o_half_screen_height" separator=" "/>
+    </xpath>
+</template>
+
+
 <template id="new_page_template_landing_0_s_cover" inherit_id="website.new_page_template_landing_0_s_cover">
     <xpath expr="//section" position="attributes">
         <!-- pb256 is defined in both theme and new page template -->
-        <attribute name="class" add="pb256" remove="pt0 pb256" separator=" "/>
+        <attribute name="class" add="pb80 pt80" remove="pt256 pb256 pt0 pb0" separator=" "/>
```
Comment (xO-Tx) [inline theme_vehicle/views/new_page_template.xml:128]: After theme customization, the default `s_cover` will have `pt0 pb0`. Why is the `pt256 pb256` removal needed here :thinking: ?

**Review by xO-Tx — CHANGES_REQUESTED** [review]
@xlu-odoo Just a small remark and it will be good to go :+1:

File: theme_vehicle/views/new_page_template.xml (hunk 3 above)
Comment (xlu-odoo) [reply theme_vehicle/views/new_page_template.xml:128]: Without that, a new page template (landing cover 0) wasn't functionning properly and test was failing because of duplicate class. I remember spending some time trying debugging at that time, and that worked... : / If you have any other ideas..

**Comment by qsm-odoo** [comment]
@robodoo delegate=xO-Tx 
@xO-Tx

**Comment by xO-Tx** [comment]
@robodoo r+
